ci/test.sh

```diff
@@ -113,7 +113,14 @@ test_template() {
     popd
 }
 
-for x in $assets_dir/*.yaml
+if [ "$GENERATE_ALL" == "true" ]
+then
+    indexPath=$assets_dir/*.yaml
+else
+    indexPath=$assets_dir/*-local.yaml
```

Comment:
I think test should always go against *-local.yaml. I think I got this wrong in the first pass.

*-local.yaml files will only exist for content that was generated.. so this should self-limit (and could correct some other problems with automatic addition of repositories)

Reply:
Just for clarification, are you saying ```indexPath``` should always be *-local.yaml regardless of the value of ```GENERATE_ALL```?